Dockerfile

```diff
@@ -27,6 +27,13 @@ RUN cargo install --path forest
 # Prod image for forest binary
 FROM debian:buster-slim
 
+# Install binary dependencies
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y curl file gcc g++ hwloc libhwloc-dev make openssh-client \
+    autoconf automake cmake libtool libcurl4 libcurl4-openssl-dev libssl-dev \
+    libelf-dev libdw-dev binutils-dev zlib1g-dev libiberty-dev wget \
+    xz-utils pkg-config python clang ocl-icd-opencl-dev ca-certificates
```

Comment:
Do you need these to run? I just want to know what specifically was wrong with the dockerfile

Reply:
Yes the binary still needs the dependencies installed or else the error `error while loading shared libraries: libcurl.so.4: cannot open shared object file: No such file or directory` will be present.

Reply:
In terms of what was specifically wrong, the previous docker image was using an outdated version, missing required deps of `hwloc libhwloc-dev ca-certificates` and the multistage build didn't have the required deps installed.

Reply:
Can definitely make the argument that a multi stage build in this circumstance is not really advantageous, happy to remove if so.

Reply:
hmm interesting, I wouldn't expect the second image would need those deps, I assume it needed hwloc? maybe we can try to narrow down exactly what's needed for this second image (doesn't have to be now)